src/renderer/npm.ts

```diff
@@ -33,22 +34,37 @@ const isUnique = (item: any, idx: number, arr: Array<any>): boolean => {
 };
 
 /**
- * Checks if npm is installed by checking if a binary
+ * Checks if package manager is installed by checking if a binary
  * with that name can be found.
  */
-export async function getIsNpmInstalled(
+export async function getIsPackageManagerInstalled(
+  packageManager: IPackageManager,
   ignoreCache?: boolean,
 ): Promise<boolean> {
-  if (isInstalled !== null && !ignoreCache) return isInstalled;
+  if (packageManager === 'npm' && isNpmInstalled !== null && !ignoreCache)
+    return isNpmInstalled;
+  if (packageManager === 'yarn' && isYarnInstalled !== null && !ignoreCache)
+    return isYarnInstalled;
 
-  const command = process.platform === 'win32' ? 'where.exe npm' : 'which npm';
+  const command =
+    process.platform === 'win32'
+      ? `where.exe ${packageManager}`
+      : `which ${packageManager}`;
```

Comment:
Perhaps you can use `hasYarn` from [`yarn-or-npm`](https://www.npmjs.com/package/yarn-or-npm) (which Electron Forge already uses).

Alternatively, perhaps use the [`which`](https://www.npmjs.com/package/which) module.